src/lib/routes/client-api/feature.test.ts

```diff
@@ -130,10 +130,10 @@ test('fetch single feature', async () => {
 
 test('support name prefix', async () => {
     expect.assertions(2);
-    await featureStrategiesStore.createFeature({ name: 'a_test1' });
-    await featureStrategiesStore.createFeature({ name: 'a_test2' });
-    await featureStrategiesStore.createFeature({ name: 'b_test1' });
-    await featureStrategiesStore.createFeature({ name: 'b_test2' });
+    await featureToggleClientStore.createFeature({ name: 'a_test1' });
+    await featureToggleClientStore.createFeature({ name: 'a_test2' });
+    await featureToggleClientStore.createFeature({ name: 'b_test1' });
+    await featureToggleClientStore.createFeature({ name: 'b_test2' });
```

Comment:
I'm confused, why is the clientStore creating features? I thought it was meant only for the /api/client endpoints?

Reply:
Wouldn't createFeature be more suited for a featureToggleAdminStore?

Reply:
I totally agree, I am even unsure if this actually works, or there might be a typo in its name.

Reply:
So this is where our fake-stores got limitations..... one fake store cannot insert data to another fake store. Suggestions?

Reply:
I will rename it to _createFeature for now and add a comment.